### ISSUE TITLE
Nextra 3.0.0-alpha.15 error

### ISSUE THREAD

**Issue by asimaranov** [shuding/nextra#2664]
When I update nextra to 3.0.0-alpha.15 or 3.0.0-alpha.14, it fails with this error. If downgraded back to 3.0.0-alpha.13 it works

```
 ✓ Collecting page data    
   Generating static pages (0/17)  [    ]TypeError: Cannot destructure property 'auth' of 'e' as it is undefined.
    at i (/Users/andrey/projects/zknoid_docs/.next/server/chunks/651.js:1:53647)
    at s (/Users/andrey/projects/zknoid_docs/.next/server/chunks/651.js:1:54336)
    at d (/Users/andrey/projects/zknoid_docs/.next/server/chunks/651.js:1:12290)
    at /Users/andrey/projects/zknoid_docs/.next/server/chunks/651.js:1:7761
    at Object.Kc [as useMemo] (/Users/andrey/projects/zknoid_docs/node_modules/react-dom/cjs/react-dom-server.browser.production.min.js:60:240)
    at exports.useMemo (/Users/andrey/projects/zknoid_docs/node_modules/react/cjs/react.production.min.js:25:208)
    at /Users/andrey/projects/zknoid_docs/.next/server/chunks/651.js:1:7677
    at Wc (/Users/andrey/projects/zknoid_docs/node_modules/react-dom/cjs/react-dom-server.browser.production.min.js:68:44)
    at Zc (/Users/andrey/projects/zknoid_docs/node_modules/react-dom/cjs/react-dom-server.browser.production.min.js:73:362)
    at Z (/Users/andrey/projects/zknoid_docs/node_modules/react-dom/cjs/react-dom-server.browser.production.min.js:76:89)

Error occurred prerendering page "/docs/business_model". Read more: https://nextjs.org/docs/messages/prerender-error
TypeError: Cannot destructure property 'auth' of 'e' as it is undefined.
    at i (/Users/andrey/projects/zknoid_docs/.next/server/chunks/651.js:1:53647)
    at s (/Users/andrey/projects/zknoid_docs/.next/server/chunks/651.js:1:54336)
    at d (/Users/andrey/projects/zknoid_docs/.next/server/chunks/651.js:1:12290)
    at /Users/andrey/projects/zknoid_docs/.next/server/chunks/651.js:1:7761
    at Object.Kc [as useMemo] (/Users/andrey/projects/zknoid_docs/node_modules/react-dom/cjs/react-dom-server.browser.production.min.js:60:240)
    at exports.useMemo (/Users/andrey/projects/zknoid_docs/node_modules/react/cjs/react.production.min.js:25:208)
    at /Users/andrey/projects/zknoid_docs/.next/server/chunks/651.js:1:7677
    at Wc (/Users/andrey/projects/zknoid_docs/node_modules/react-dom/cjs/react-dom-server.browser.production.min.js:68:44)
    at Zc (/Users/andrey/projects/zknoid_docs/node_modules/react-dom/cjs/react-dom-server.browser.production.min.js:73:362)
    at Z (/Users/andrey/projects/zknoid_docs/node_modules/react-dom/cjs/react-dom-server.browser.production.min.js:76:89)
 ✓ Generating static pages (17/17) 

> Export encountered errors on following paths:
        /docs/business_model
error Command failed with exit code 1.
```

**Comment by asimaranov** [comment]
The problem was that one page file was not created that declared in meta

**Comment by dimaMachina** [comment]
@asimaranov Yeah, i will think where I can add a helpful human error instead of TypeError

**Comment by dimaMachina** [comment]
try to declared not found pages in `nextra-theme-docs/nextra` `3.1.0-canary.0`

**Comment by dimaMachina** [comment]
fixed in nextra 3.1.0
